pkg/reconcilermanager/controllers/garbage_collector.go

```diff
@@ -91,13 +92,17 @@ func (r *reconcilerBase) cleanup(ctx context.Context, obj client.Object) error {
 	return nil
 }
 
-func (r *RepoSyncReconciler) deleteSecrets(ctx context.Context, reconcilerRef types.NamespacedName) error {
+func (r *RepoSyncReconciler) deleteSecrets(ctx context.Context, reconcilerRef types.NamespacedName, currentSecrets []string) error {
```

Comment:
Or `exceptions... types.NamespacedName` so you don't need to add the `nil` in the normal case.

Reply:
I think `types.NamespacedName` is redundant here because it's already assumed the Secrets are in the same Namespace as the reconciler (see the `List` below). But the variadic arg works for me